utkdigitalinitiatives/utk_isl_xml_forms
# Address TRAC-424

**JIRA Ticket**: [TRAC-424](https://)

# What does this Pull Request do?
Adds and updates template processing for mods:name[@authority='orcid'][@valueURI].

# What's new?
* If a @valueURI is empty, strip all of the attributes except for @type.
* If a @valueURI is NOT empty and does NOT start with 'http:orcid.org', then prepend 'http://orcid.org/' to @valueURI
* If a @valueURI is NOT empty and starts with 'http:orcid.org', copy @valueURI

# How should this be tested?
Clone and scp the UTK_ir_etd_post_process transform to VM:/var/www/drupal/sites/all/modules/islandora_xml_forms/builder/self_transforms. Submit and verify data, then edit the MODS datastream and verify.

# Additional Notes:
This still doesn't perform any 'validation' on the orcid value.

# Interested parties
@markpbaggett
